src/MonoTorrent/MonoTorrent.Client/Managers/InactivePeerManager.cs

```diff
@@ -103,6 +108,11 @@ internal void TimePassed ()
                         }
                     }
                 }
+                }
+                catch (Exception ex)
+                {
+                    Debug.WriteLine("TimePassed Error " + ex.Message);
```

Comment:
Instead of doing Debug.WriteLine here and in other places, could you create a class logger as static readonly Logger logger = Logger.Create (); and then log it as logger.Debug(...

Reply:
Definitely +1 on the Logger.Create change :)